test/bdd/fixtures/discovery-config/sidetree-fabric/config-data/peer1-org1.trustbloc.local.json

```diff
@@ -0,0 +1,12 @@
+{
+  "domain": "peer1-org1.trustbloc.local",
```

Comment:
I don’t understand. Is this some special test data?

There should be org1.trustbloc.local for the deployment.

Reply:
we don't have load balancer locally we use peer container name as domain

Reply:
done